server/models/remote_provider.go

```diff
@@ -1810,6 +1810,12 @@ func (l *RemoteProvider) GetCatalogMesheryPatterns(tokenString string, page, pag
 			q.Add("userid", user)
 		}
 	}
+
+	if len(workspaceID) > 0 {
+		for _, workspace := range workspaceID {
+			q.Add("workspaceID", workspace)
+		}
+	}
```

Comment:
We don't need to pass multiple workspace ids in request as remote provider only accept single workspace id at a time